enterprise/backend/test/metabase_enterprise/audit_app/api/collection_test.clj

```diff
@@ -15,12 +22,12 @@
         (testing "Instance Analytics Collection should be the last collection."
           (testing "GET /api/collection"
             (is (= "instance-analytics"
-                   (->> (mt/user-http-request :rasta :get 200 "collection")
+                   (->> (mt/user-http-request :crowberto :get 200 "collection")
```

Comment:
Swapping to admin user as non admin default shouldn't be able to see metabase analytics